pkg/3scale/amp/component/system.go

```diff
@@ -1215,7 +1216,7 @@ func (system *System) getSystemServiceDiscoveryData() string {
 `
 }
 
-func (system *System) SphinxDeploymentConfig() *appsv1.DeploymentConfig {
+func (system *System) SphinxDeploymentConfig(priorityClassName string) *appsv1.DeploymentConfig {
```

Comment:
are we missing priorityclassname below?

Reply:
Thank you for comment, please see reply above